src/Components/App.js

```diff
@@ -3,26 +3,42 @@ import '../CSS_styling/App.css';
 import movieData from '../movie-data';
 import Movies from './Movies';
 import NavBar from './NavBar';
+import Details from './Details'
 
 class App extends Component {
   constructor() {
    super()
    this.state = {
      movieInfo: movieData.movies
    }
-   console.log(this.state.movieInfo)
+   console.log("this.state.movieInfo here---", this.state.movieInfo)
+  }
+
+  findDetails = (event) => {
```

Comment:
It was incredible working through troubleshooting this, you have a great knack for debugging.